cmd/subcommands/staking.go

```diff
@@ -70,7 +70,10 @@ var (
 	errInvalidDescFieldDetails         = errors.New("exceeds maximum length of 280 characters for description field details")
 )
 
-func getNextNonce(addr oneAddress, messenger rpc.T) uint64 {
+func getNextNonce(addr oneAddress, messenger rpc.T, nonce int64) uint64 {
+	if nonce != -1 {
```

Comment:
what happens when people give wacky other values for nonce from cli input, like -2.